packages/cli/src/oclif/commands/env/set.js

```diff
@@ -45,21 +45,30 @@ class SetEnvCommand extends BaseCommand {
     }
 
     const url = `/apps/${app.id}/versions/${version}/multi-environment`;
+    const requestOptions = {
+      body: payload,
+      method: 'POST',
+    };
+
+    if (this.flags.force) {
+      requestOptions.extraHeaders = {
+        'X-Force-Env-Var-Update': 'true',
```

Comment:
hmmm I don't see this header referenced in the `/multi-environment/` [endpoint](https://gitlab.com/zapier/zapier/-/blob/main/web/backend/zapier/app_platform/developer_cli/api/resources.py#L2462) 🤔 or the Mixin classes it's a child of, are the headers directly passed to the lambda pools or am I looking at the wrong place?

Reply:
I believe [this MR](https://gitlab.com/zapier/zapier/-/merge_requests/60723) is the corresponding backend change.